.travis.yml

```diff
@@ -1,23 +1,25 @@
 language: php
 
-php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-  - nightly
-  - hhvm
-
 matrix:
+  fast_finish: true
   allow_failures:
     - php: nightly
+  include:
+    - php: 5.3
+    - php: 5.4
+    - php: 5.5
+    - php: 5.6
+    - php: 7.0
+    - php: 7.1
+      env:
+        - TEST_COVERAGE=true
```

Comment:
> Is this effectively saying to only run test coverage and report that to Coveralls only under PHP 7.1?

Yes, exactly :-)

> Should coverage not be reported under every version, to make sure that all lines are run during tests on all versions of PHP?

I can't think of any reason why the PHP version should affect the coverage, if there aren't different code paths, but maybe I'm missing something?